timemachine/fe/rbfe.py

```diff
@@ -775,6 +702,13 @@ class Edge(NamedTuple):
     mol_b_name: str
     metadata: Dict[str, Any]
 
+    def __str__(self):
+        name = f"{self.mol_a_name} -> {self.mol_b_name} (kJ/mol)"
```

Comment:
nit: seems like it might get a little confusing to hardcode kJ/mol here. Also it feels a little bit awkward to me to have the `__str__` method depend on specific metadata entries (I'd originally intended this to be a place for ad-hoc data that may or may not be present). I like the idea in general; maybe one fix could be to move `exp_ddg`, `fep_ddg`, etc. out of metadata to (optional) attributes?

Reply:
Not sure -- reverting for now https://github.com/proteneer/timemachine/pull/936/commits/8f33e440a2184e3fcab5a7480095d6fbb5d91fe0